app-emulation/fs-uae-launcher/patches/fs_uae_launcher-2.8.3.patchset

```diff
@@ -0,0 +1,49 @@
+From 5c22ad1fae1b33eb3536922a07573e65534c70bb Mon Sep 17 00:00:00 2001
+From: Chris Moore <chris@mooreonline.org>
+Date: Tue, 12 Sep 2017 20:02:46 +0100
+Subject: Stop linking the program to another directory
+
+
+diff --git a/Makefile b/Makefile
+index f33d513..3cc0053 100644
+--- a/Makefile
++++ b/Makefile
+@@ -106,12 +106,10 @@ ifeq ($(DESTDIR),)
+ 	python3 setup.py install	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
+ 	install -d $(DESTDIR)$(prefix)/bin
+ 	rm -f $(DESTDIR)$(prefix)/bin/fs-uae-launcher
+-	ln -s ../share/fs-uae-launcher/fs-uae-launcher	 $(DESTDIR)$(prefix)/bin/fs-uae-launcher
+ else
+ 	python3 setup.py install	 --root=$(DESTDIR)	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
+ 	install -d $(DESTDIR)$(prefix)/bin
+ 	rm -f $(DESTDIR)$(prefix)/bin/fs-uae-launcher
+-	ln -s ../share/fs-uae-launcher/fs-uae-launcher	 $(DESTDIR)$(prefix)/bin/fs-uae-launcher
+ endif
+ 
+ install-data: mo
+-- 
+2.13.1
+
+
+From d6c8205fdc64ed0af5b6a42f36026b59cf8d8bc6 Mon Sep 17 00:00:00 2001
+From: Chris Moore <chris@mooreonline.org>
+Date: Tue, 12 Sep 2017 20:07:32 +0100
+Subject: Create files in local directories
+
+
+diff --git a/Makefile b/Makefile
+index 3cc0053..977a385 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,7 +4,7 @@ dist_name = fs-uae-launcher-$(version)
+ dist_dir := $(build_dir)/$(dist_name)
+ 
+ DESTDIR :=
+-prefix := /usr/local
++prefix := usr/local
```

Comment:
What does this change do?

Reply:
Installs the files locally before they are copied to the correct locations in the recipe.

Reply:
Couldn't you use prefix=@prefix@ or in the recipe "make PREFIX=$prefix"?